functions/venue.js

```diff
@@ -203,8 +207,13 @@ const createVenueData = (data, context) => {
     venueData.mapBackgroundImageUrl = data.mapBackgroundImageUrl;
   }
 
-  if (data.template === VenueTemplate.audience) {
-    venueData.showReactions = data.showReactions;
+  // @debt showReactions and showShoutouts should be toggleable for anything in HAS_REACTIONS_TEMPLATES
+  if (HAS_REACTIONS_TEMPLATES.includes(data.template)) {
+    venueData.showReactions =
+      typeof data.showReactions === "boolean" ? data.showReactions : true;
+
+    venueData.showShoutouts =
+      typeof data.showShoutouts === "boolean" ? data.showShoutouts : true;
```

Comment:
This should default showReactions and showShoutouts to true (but let me know if I'm wrong)

Reply:
we might just transform it to `venueData.showShoutouts = !!data.showShoutouts || true`
correct me if I'm wrong

Reply:
won't that make `undefined` or `null` false when we want it to be true?

Reply:
@alexey-petrov nope

@malaikaaiyar your initial way is correct